src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -231,6 +239,14 @@ class InflictorWithValue extends React.Component {
               {value > 0 && value}
             </div>
           }
+            <div className='textOverlay'>
```

Comment:
think we want to not render this div at all, unless we have charges content to put here. That should fix the issue with backpack items (unless there's also a weird interaction between backpack items with charges)

Reply:
we should be able to do just chargeOverlay right?  textOverlay didn't exist before this change?